### ISSUE TITLE
Cannot find module 'C:\test\jestTest\node_modules\@davinkevin\jest\src\ng-add\index'

### ISSUE THREAD

**Issue by SanderElias** [davinkevin/jest#2]
The following sequence:

```
ng new jestTest
cd jestTest
ng add @davinkevin/jest
```

gives me the following result:

![image](https://user-images.githubusercontent.com/1249083/44453360-28845100-a5f9-11e8-898a-155ae1286e67.png)

ng version:
```

Angular CLI: 6.1.4
Node: 8.11.1
OS: win32 x64
Angular: 6.1.3
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.7.4
@angular-devkit/build-angular     0.7.4
@angular-devkit/build-optimizer   0.7.4
@angular-devkit/build-webpack     0.7.4
@angular-devkit/core              0.7.4
@angular-devkit/schematics        0.7.4
@angular/cli                      6.1.4
@ngtools/json-schema              1.1.0
@ngtools/webpack                  6.1.4
@schematics/angular               0.7.4
@schematics/update                0.7.4
rxjs                              6.2.2
typescript                        2.7.2
webpack                           4.9.2
```

Did I miss something?

**Comment by davinkevin** [comment]
I fixed the problem, seems to be ok on v0.0.9 : 

![image](https://user-images.githubusercontent.com/1970922/44503815-9df32e80-a698-11e8-9bbd-e84a833c7242.png)

Let me know if there is other problem

**Comment by davinkevin** [comment]
For now, you have to add `"module": "commonjs",` in the `tsconfig.spec.json` to make things work. I will upgrade the schematics to do this for you as soon as possible

**Comment by davinkevin** [comment]
Should be fixed in `0.0.10`

**Comment by SanderElias** [comment]
Version "0.0.10" fixes the issue, also for CLI 6.2.0-beta.2 and Angular 6.1.4

**Comment by davinkevin** [comment]
Thanks
